### PR TITLE
add fatal error log file

### PR DESCRIPTION
**Pull request by Gl4di4torRr**
I was reading over the documentation and noticed this override is not in the documentation. Let me know if y'all want this in the official docs.

### CODE REVIEW

**Comment by Gl4di4torRr** [comment]
this relates to https://github.com/openshift/jenkins/blob/master/README.md

**Comment by Gl4di4torRr** [comment]
@gabemontero @bparees one of you probably know if you would like this override in the docs. sorry to tag you like this... i opened https://github.com/openshift/openshift-docs/issues/11809 so i don't repeat my behavior :)

File: using_images/other_images/jenkins.adoc [hunk 1]
```diff
@@ -270,6 +270,10 @@ that extends this image and updates plugins in the custom image after
 the initial startup, by default they are not copied over, unless you set this
 environment variable to `true`.
 
+* `ENABLE_FATAL_ERROR_LOG_FILE` (default: `false`) 
++
+When running this image with an OpenShift persistent volume claim for the Jenkins config directory, this environment variable allows the fatal error log file to persist if a fatal error occurs. The fatal error file will be located at `/var/lib/jenkins/logs`.
```
Comment (bparees) [inline using_images/other_images/jenkins.adoc]: s/OpenShift//

**Review by bparees — DISMISSED** [review]

File: using_images/other_images/jenkins.adoc (hunk 1 above)
Comment (Gl4di4torRr) [reply using_images/other_images/jenkins.adoc]: @bparees sorry, what are you wanting changed?

Comment (bparees) [reply using_images/other_images/jenkins.adoc]: delete the word openshift

Comment (bparees) [reply using_images/other_images/jenkins.adoc]: (and change the "an" to an "a")

**Comment by Gl4di4torRr** [comment]
@bparees completed the change

**Comment by bparees** [comment]
lgtm.
@openshift/team-documentation ptal

**Comment by bparees** [comment]
@gabemontero what versions did this flag go into?

**Comment by gabemontero** [comment]
@bparees @openshift/team-documentation this is 3.11 only

**Comment by vikram-redhat** [comment]
@gaurav-nelson - PTAL.

**Pull request merged**